content/actions/reference/environment-variables.md

```diff
@@ -64,7 +64,7 @@ We strongly recommend that actions use environment variables to access the files
 | `GITHUB_REPOSITORY` | The owner and repository name. For example, `octocat/Hello-World`. |
 | `GITHUB_EVENT_NAME` | The name of the webhook event that triggered the workflow. |
 | `GITHUB_EVENT_PATH` | The path of the file with the complete webhook event payload. For example, `/github/workflow/event.json`. |
-| `GITHUB_WORKSPACE` | The {% data variables.product.prodname_dotcom %} workspace directory path. The workspace directory is a copy of your repository if your workflow uses the [actions/checkout](https://github.com/actions/checkout) action. If you don't use the `actions/checkout` action, the directory will be empty. For example, `/home/runner/work/my-repo-name/my-repo-name`. |
+| `GITHUB_WORKSPACE` | The {% data variables.product.prodname_dotcom %} workspace directory path, initially empty. The [actions/checkout](https://github.com/actions/checkout) action will check out files, by default a copy of your repository, within this directory. For example, `/home/runner/work/my-repo-name/my-repo-name`. |
```

Comment:
Fwiw, the `for example` part of this text is actually confusing.

I believe that, technically, it's saying `GITHUB_WORKSPACE` = `/home/runner/work/my-repo-name/my-repo-name` _and_ if you did a default actions/checkout of the `github.com/my-org/my-repo`, it would happen to populate that directory with those files.

As a total aside, it would be really helpful is some hint about why the workspace has two layers of `my-repo-name`. (Or at least a very good hint that the first one is not a typo for `my-org-name`.)

I'm sure it's out of style, but having the text in two paragraphs as:

> The {% data variables.product.prodname_dotcom %} workspace directory path, initially empty.
> For example, `/home/runner/work/my-repo-name/my-repo-name`.
>
> The [actions/checkout](https://github.com/actions/checkout) action will check out files, by default a copy of your repository, within this directory.

would be even less confusing.

That said, I believe that the change I'm currently proposing is still an improvement over the current text, and don't want this comment to distract from that point.

Reply:
I like your proposal to reorder the text:

```suggestion
| `GITHUB_WORKSPACE` | The {% data variables.product.prodname_dotcom %} workspace directory path, initially empty. For example, `/home/runner/work/my-repo-name/my-repo-name`. The [actions/checkout](https://github.com/actions/checkout) action will check out files, by default a copy of your repository, within this directory. |
```